src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `${minutes}m ago` to `${minutes}m` and similar changes in other lines make the time representation more concise. However, it might lead to confusion for users as the 'ago' context is lost. Consider keeping 'ago' or providing a clear indication of past time elsewhere in the UI.
```diff
- return minutes === 0 ? '1m' : `${minutes}m`;
+ return minutes === 0 ? '1m ago' : `${minutes}m ago`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->